saleor/dashboard/product/utils.py

```diff
@@ -0,0 +1,11 @@
+from ...extensions.manager import ExtensionsManager, get_extensions_manager
+from ...product.models import Product
+
+
+def get_product_tax_rate(product: Product, manager: ExtensionsManager = None) -> str:
```

Comment:
Shouldn't we use the same logic in the API as well? If this was the case, I think we could have this logic in the manager's `get_tax_code_from_object_meta` method. @korycins what do you think?

Reply:
In the API we only retrieve the value once (resolver). I am not sure if there would be any other use case. That's why I put it into the dashboard utilities for now, so it can be removed safely with the dashboard.

Reply:
In the dashboard, we're getting the tax rate from product and if it's unknown,  then we're getting it from the product type. Don't we have to use the same logic over there as well? If not then fine, but I thought it should be reused.